src/frontend/planner_test/tests/testdata/sysinfo_funcs.yaml

```
@@ -14,3 +14,23 @@
   batch_plan: |
     BatchProject { exprs: ['root':Varchar] }
     └─BatchValues { rows: [[]] }
+- sql: |
+    select current_schemas(true);
+  batch_plan: |
+    BatchProject { exprs: [ARRAY[pg_catalog, public]:List { datatype: Varchar }] }
+    └─BatchValues { rows: [[]] }
+- sql: |
+    select current_schemas(false);
+  batch_plan: |
+    BatchProject { exprs: [ARRAY[public]:List { datatype: Varchar }] }
+    └─BatchValues { rows: [[]] }
+- sql: |
+    select current_schemas(null);
+  batch_plan: |
+    BatchProject { exprs: [null:List { datatype: Varchar }] }
+    └─BatchValues { rows: [[]] }
+- sql: |
+    select current_schemas(true and false);
+  binder_error: |-
+    Feature is not yet implemented: Only boolean literals are supported.
```

```suggestion
    Feature is not yet implemented: Only boolean literals are supported in `current_schemas`.
```

This error message is not clear when used in a large query.